packages/react-native/README.md

```diff
@@ -1,25 +1,83 @@
-# Babel-based transpiling environment for React Native components
+# React Native Compiler
 
-Bit build enviroment for transpiling React Native components using Bit.
+A ReactNative component compiler for [Bit](https://github.com/teambit/bit).
+This compiler follows our best practices guide for compiling shared components.
 
 ## How to use?
 
-Import the environment.
+In order to run this extension your must have a [bit workspace](https://docs.bit.dev/docs/concepts#bit-workspace) with at least one component defined, for more information on how to build please read the [docs](https://docs.bit.dev/docs/building-components) section on the bit website. TL;DR version:
 
-```bash
-bit import bit.envs/compilers/react-native -c
+Install the React Native compiler
+
+```
+$ bit import bit.envs/compilers/react-native -c
 ```
 
-Then build using [bit build](https://docs.bitsrc.io/docs/cli-build.html).
+Then, you can simply build the component using `bit build`
 
-```bash
-bit build
 ```
+$ bit build
+```
+
+## Features
+
+- **Compile components in isolation** - Compiling shared components in true isolation is both challenging and important. It helps to find reusability issues quickly and early throughout the development process. It makes sure your components will compile identically anywhere and without side-effects. Read more about isolation [here](https://docs.bit.dev/docs/ext-concepts.html#what-is-an-isolated-component-environment).
+
+- **Configuration** - This compiler is configured according to best practices for shared components compiling learned from our experience. For general concepts on how to approach shared components compilation and testing, please see the main [readme](https://github.com/teambit/envs). Feel free to open an issue or submit a PR if you think something should be different or there is a bug in the implementation.
```

Comment:
Which best practices are specific to react-native

Reply:
Using the preset https://www.npmjs.com/package/metro-react-native-babel-preset and not the old one https://www.npmjs.com/package/babel-preset-react-native.
The new one is working with Babel 7 and includes all the babel plugins and also supports optional chaining.

Reply:
Something from this should be in the doc